packages/sdk/ts/utils/types.ts

```diff
@@ -593,3 +599,152 @@ export interface IExtractAllVksArgs {
    */
   tallyVotesZkeyPath?: string;
 }
+
+/**
+ * Interface that represents Verification key
+ */
+export interface ISnarkJSVerificationKey {
+  protocol: BigNumberish;
+  curve: BigNumberish;
+  nPublic: BigNumberish;
+  vk_alpha_1: BigNumberish[];
+  vk_beta_2: BigNumberish[][];
+  vk_gamma_2: BigNumberish[][];
+  vk_delta_2: BigNumberish[][];
+  vk_alphabeta_12: BigNumberish[][][];
+  IC: BigNumberish[][];
+}
+
+/**
+ * Inputs for circuit PollJoining
+ */
+export interface IPollJoiningInputs {
+  privKey: bigint;
+  pollPubKey: bigint[][];
+  stateLeaf: bigint[];
+  siblings: bigint[][];
+  indices: bigint[];
+  nullifier: bigint;
+  credits: bigint;
+  stateRoot: bigint;
+  actualStateTreeDepth: bigint;
+  pollId: bigint;
```

Comment:
Do we need docs for the props for all the interfaces here?

Reply:
we should, but for now I've only been adding to new ones, and others copy pasting. Probably keeping adding docs to all of them as last step